desktop/php/scenario.php

```diff
@@ -371,7 +371,7 @@ function jeedom_displayScenarioGroup($_group = '', $_index = -1) {
 
 				<div id="md_addElement" class="jeeDialog jeeDialogPrompt" style="display:none;">
 					<div class="jeeDialogTitle">
-						<span class="title">{{Ajouter un bloc}}</span><button class="btClose" type="button">×</button>
+						<span class="title">{{Ajouter un bloc}}</span><button class="btClose" id="bt_crossElementSave" type="button"></button>
```

Comment:
Consider using more descriptive id names for elements. For instance, instead of 'bt_crossElementSave', you could use 'btn_closeModalDialog'. This would make the code more readable and maintainable. [medium]

Reply:
I literally used the same semantics as existing close and OK button here:
https://github.com/jeedom/core/blob/0f13b9bc1a3a1d095b617dc3827de737c2a432d1/desktop/php/scenario.php#L418-L419